doc/man1/openssl-s_server.pod.in

```diff
@@ -599,6 +601,17 @@ further information).
 There are several known bugs in SSL and TLS implementations. Adding this
 option enables various workarounds.
 
+=item B<-cert_comp>
+
+Enables support for SSL/TLS certificate compression, same as clearing
```

Comment:
TLS 1.3 (same applies to section below).

Reply:
Fixed